src/scala/ly/stealth/mesos/kafka/Executor.scala

```diff
@@ -92,6 +95,25 @@ object Executor extends org.apache.mesos.Executor {
       }
     }
 
+    def startCollectingMetrics(): Unit = {
+      new Thread {
+        setName("BrokerMetrics")
+        def send(driver: ExecutorDriver, metrics: Broker.Metrics): Unit = {
+          driver.sendFrameworkMessage(JSONObject(Map("metrics" -> metrics.toJson)).toString().getBytes)
+        }
+        override def run(): Unit = {
+          while (true) {
+            try {
+              BrokerServer.Metrics.collect.foreach(metrics => send(driver, metrics))
+              Thread.sleep(5000L)
+            } catch {
+              case e: Throwable => logger.warn("", e)
```

Comment:
We should start using the kafka logging class https://github.com/apache/kafka/blob/trunk/core/src/main/scala/kafka/utils/Logging.scala it has some nice ways to make sure we squeak out performance. Separate issue just thinking about it here we should keep consistency overall and refactor later when need be. That may or may not also apply to the new Thread continued approach it seems the more we add of them the harder it will be to manage them when running.